### PR TITLE
Added tests for default named class export

### PR DESCRIPTION
**Pull request by nickshevr**
This tests is regression for cases from:
https://github.com/lttb/reselector/issues/16

### CODE REVIEW

File: src/tests/__snapshots__/babel.spec.js.snap [hunk 1]
```diff
@@ -122,6 +142,32 @@ const B = () => _react.default.createElement(_react.Fragment, null, _react.defau
...
+exports.default = void 0;
+
+var _react = _interopRequireWildcard(require(\\"react\\"));
+
+function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) { var desc = Object.defineProperty && Object.getOwnPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : {}; if (desc.get || desc.set) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } } newObj.default = obj; return newObj; } }
+
+class DefaultClass extends _react.Component {
+  render() {
+    return _react.default.createElement(\\"div\\", {
+      \\"data-7d3c8cda\\": true
```
Comment (nickshevr) [inline src/tests/__snapshots__/babel.spec.js.snap]: @a-omsk it added testId for dom-elem

**Comment by nickshevr** [comment]
So we should test `resolve` function?

File: src/tests/__snapshots__/babel.spec.js.snap [hunk 2]
```diff
@@ -122,6 +142,33 @@ const B = () => _react.default.createElement(_react.Fragment, null, _react.defau
...
+    }, this.props.children);
+  }
+
+}
+
+DefaultClass.testId = \\"7d3c8cda\\";
```
Comment (nickshevr) [inline src/tests/__snapshots__/babel.spec.js.snap]: This property assignment looks good also

File: src/tests/__snapshots__/babel.spec.js.snap [hunk 3]
```diff
@@ -147,6 +194,35 @@ if (module.exports) {
...
+      \\"data-51b7b868\\": true
+    }, this.props.children);
+  }
+
+}
+
+exports.default = DefaultClass;
+
+if (module.exports) {
+  module.exports.default.testId = \\"51b7b868\\";
```
Comment (nickshevr) [inline src/tests/__snapshots__/babel.spec.js.snap]: Maybe this one doesn't work well with `resolve` logic?
@lttb

Comment (lttb) [reply src/tests/__snapshots__/babel.spec.js.snap]: what do you mean? `resolve` logic does not depend on transpiled files

**Pull request merged**